scripts/fixup_headers.py

```diff
@@ -0,0 +1,187 @@
+#!/usr/bin/env python3
+
+# This script checks and can optionally update FENIX source files.
+# You should always run this script without the "-u" option
+# first to make sure there is a clean dry run of the files that should
+# be updated
+# This is based on a script of the same name in the MOOSE Framework:
+# https://github.com/idaholab/moose/blob/master/framework/scripts/fixup_headers.py
+
+import os, string, re, shutil
+from optparse import OptionParser
+
+global_dir_ignores = ['contrib', '.svn', '.git', 'crane', 'moose', 'squirrel']
+global_file_ignores = ['moosedocs.py']
```

Comment:
```suggestion
global_file_ignores = ['moosedocs.py', 'fixup_headers.py']
```

Let's also make sure it ignores itself, if it doesn't already. If it ignored this file on first run, please disregard this adjustment.

Reply:
The file ignores itself.